src/features/hotspots/details/HotspotDetailChart.tsx

```diff
@@ -143,17 +150,18 @@ const HotspotDetailChart = ({
             </Box>
             <Box width="60%">
               <ChartContainer
-                height={75}
+                height={90}
                 data={data}
                 onFocus={onFocus}
                 showXAxisLabel={false}
-                upColor={color}
+                upColor={change >= 0 ? greenOnline : purpleMain}
                 downColor={grayLight}
                 labelColor={black}
               />
-              <Text variant="body3" color="black" paddingTop="xs">
+              {/* TODO removing this for now to fit the design better */}
+              {/* <Text variant="body3" color="black" paddingTop="xs">
                 {focusedData ? focusedData.day : ' '}
```

Comment:
Can we put it somewhere else? It is nice to see the day

Reply:
I'll make a design ticket for it. having that gap there at the bottom was definitely throwing the alignment off though